WallsAndHoles/tile.cpp

```diff
@@ -41,6 +41,12 @@ QVector2D Tile::position() const
         return mRelativePosition + mTileTemplate->position();
 }
 
+const SimpleTexturedMaterial &Tile::material() const
+{
+    Q_ASSERT(hasTileTemplate());
```

Comment:
Not sure about this assert, maybe we want it to be safe to quarry an empty tile? 
Fine for now